static/js/modules/helpers.js

```diff
@@ -117,6 +117,19 @@ Handlebars.registerHelper('electionUrl', function(year, options) {
   return new Handlebars.SafeString(url);
 });
 
+Handlebars.registerHelper('capitalize', function(value) {
```

Comment:
I originally used it instead of `convertBoolean` and then decided to keep it in as it might be useful down the road and I already wrote it. Should we remove it?